spec.md

```diff
@@ -2387,6 +2389,58 @@ module DUT :
   connect y, p
 ```
 
+## Property Assignments
+
+Connections between property typed expressions (see [@sec:property-types]) are
+not supported in the `connect`{.firrtl} statement (see [@sec:connects]).
+
+Instead, property typed expressions are assigned with the `propassign`{.firrtl}
+statement.
+
+Property typed expressions have the normal rules for flow (see [@sec:flows]),
+but otherwise use a stricter, simpler algorithm than `connect`{.firrtl}. In
+order for a property assignment to be legal, the following conditions must hold:
+
+1. The left-hand and right-hand side expressions must be references (see
+   [@sec:references]).
+
+2. The left-hand and right-hand side expressions must be of property types.
+
+3. The types of the left-hand and right-hand side expressions must be the same.
+
+4. The flow of the left-hand side expression must be sink.
+
+5. The flow of the right-hand side expression must be source.
+
+6. The left-hand side expression must be used as the left-hand side in at most
+   one property assignment.
```

Comment:
The language with "must" and "at most" are slightly at odds.  I would think it would either be "may be used in at most one" or "must be used in exactly one".  The former doesn't discuss initialization checking while the latter includes initialization checking.

Reply:
Yeah, I think I would say "may be used in at most one". I am not exactly sure yet if the spec should get into initialization checking for properties or leave it open. I am leaning toward the weaker guarantee for now, and I think "may" would be the right choice if that makes sense.

Reply:
This now says "may be used in at most one". I also updated the item below from "may not" to "must not".